Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -767,7 +768,7 @@ def merge
               all_bottle_hash = { formula_name => all_bottle_formula_hash }
 
               puts "Copying #{filename} to #{all_filename}" if args.verbose?
-              FileUtils.cp filename.to_s, all_filename.to_s
+              Utils::Cp.copy filename.to_s, all_filename.to_s
```

Comment:
Is this definitely needed here? I thought later down you said it is only needed for copying casks? 

Would be good to use this method in the fewer possible places rather than the most possible places.